queryads.php

```diff
@@ -11,14 +11,28 @@
 <div class="page-header">
     <h1>Find Blocked Domain In Lists</h1>
 </div>
+
 <!-- Domain Input -->
-<div class="form-group input-group">
-    <input id="domain" type="url" class="form-control" placeholder="Domain to look for (example.com or sub.example.com)" autocomplete="off" spellcheck="false" autocapitalize="none" autocorrect="off">
-	<input id="quiet" type="hidden" value="no">
-    <span class="input-group-btn">
-        <button type="button" id="btnSearch" class="btn btn-default">Search partial match</button>
-        <button type="button" id="btnSearchExact" class="btn btn-default">Search exact match</button>
-    </span>
+<div class="visible-xs-block visible-sm-block">
+    <div class="form-group input-group-block">
+        <input id="domain" type="url" class="form-control" placeholder="Domain to look for (example.com or sub.example.com)" autocomplete="off" spellcheck="false" autocapitalize="none" autocorrect="off" style="margin-bottom: 5px">
```

Comment:
It's bad practice to use the same `id` for multiple elements. The browser will refer to the wrong element's value.
You could numberize them and then get the input values by refering to them relative from the clicked button.
But then the enter key action still needs be assigned correctly.

Reply:
I know it's bad practice, I just did this in a hurry, because, well, Bootstrap 3.x is limited.

I'm gonna drop the patch since I don't have the time to fix it properly myself. If anyone wants they are welcome to tackle this.